docs/learn/learn-guides-staking.md

```diff
@@ -87,14 +87,14 @@ nominate like you would normally using a nominator account.
 
 ![Nominate validators](../assets/staking/Nomination-Pools-5.png)
 
-The `state-toggler` can update the pool’s state to blocked through `setState` extrinsic and kick
-members by calling `unbond` and `withdrawUnbonded`. (The state can also be toggled back to open).
+The `bouncer` can update the pool’s state to blocked through `setState` extrinsic and kick members
```

Comment:
Might be worth adding that the `root` role can also do this (and destroying in the next section).